src/components/app/routes/queries/enterpriseLearner.js

```diff
@@ -36,7 +110,10 @@ const fetchEnterpriseLearnerData = async (username, enterpriseSlug, options = {}
     page: 1,
   });
   const url = `${enterpriseLearnerUrl}?${queryParams.toString()}`;
-  const linkedEnterpriseCustomersUsers = await fetchData(url);
+  const {
+    results: linkedEnterpriseCustomersUsers,
+    enterpriseFeatures,
```

Comment:
Technically dependent on https://github.com/openedx/edx-enterprise/pull/2037, but won't break without it either.